test/sim/moves/encore.js

```diff
@@ -30,11 +30,11 @@ describe('Encore', function () {
 		// If a user's previous move was Focus Punch and it is Encored into Focus Punch while attempting to
 		// execute the move, the regular "you can't be hit" effect for Focus Punch will be enforced.
 		battle.makeChoices('move focuspunch 1, move teleport', 'move encore 1, move extremespeed 1');
-		assert.equal(battle.p2.active[0].hp, hp);
+		assert.strictEqual(battle.p2.active[0].hp, hp);
```

Comment:
nooo never use `strictEqual` don't fall for it....

Reply:
ope, I used that in all of these tests because VS Code was showing assert.equal() was deprecated and said to use strictEqual instead
![image](https://user-images.githubusercontent.com/23667022/95817184-7a5f2800-0ce6-11eb-802b-d28e5051cf47.png)


should I keep using assert.equal(), or would you rather me do like assert(value=== anotherValue)? I don't really like the latter since it doesn't show the two comparisons

Reply:
Yeah, that's a VS Code bug that was fixed 4 days ago and a new version with the fix hasn't been published yet. You'll just have to deal with it being crossed out until the fix is published. :(

I've published #7515, which should make it clearer what to do here.

Reply:
Yes, thank you!